tests/foreman/api/test_convert2rhel.py

```diff
@@ -241,11 +260,7 @@ def version(request):
 
 
 @pytest.mark.e2e
-@pytest.mark.parametrize(
-    "version",
-    ['oracle7', 'oracle8'],
-    indirect=True,
-)
+@pytest.mark.parametrize('version', ['oracle7', 'oracle8'], indirect=True)
```

Comment:
could we add a `c2r` section to `supportability.yaml` and take the parameter list (of the supported versions) from there?

```
supportability:
  convert2rhel:
    oel:
      versions: [7,8]
```

Reply:
Good point, currently we're just testing C2R with small matrix of centos and oracle with v7 and v8, but I think when we include other distros to our C2R testing in future, then we can consider defining this in supportability.yml